assets/js/components/ExecutionResults/ExecutionResults.jsx

```diff
@@ -138,32 +136,57 @@ function ExecutionResults({
             key={hostID}
             hostname={hostnames.find(({ id }) => hostID === id)?.hostname}
           >
-            {filterChecks(checks, predicates).map((checkID) => {
-              const { health, error, expectations, failedExpectations } =
-                getCheckHealthByAgent(checkResults, checkID, hostID);
+            {checks
+              .map((checkID) => {
+                const { health, error, expectations, failedExpectations } =
+                  getCheckHealthByAgent(checkResults, checkID, hostID);
 
-              const label = getLabel(
-                executionData?.status,
-                health,
-                error,
-                expectations,
-                failedExpectations
-              );
-              return (
-                <CheckResult
-                  key={checkID}
-                  checkId={checkID}
-                  description={getCheckDescription(catalog, checkID)}
-                  executionState={executionData?.status}
-                  health={health}
-                  label={label}
-                  onClick={() => {
-                    setModalOpen(true);
-                    setSelectedCheck(checkID);
-                  }}
-                />
-              );
-            })}
+                return {
+                  checkID,
+                  error,
+                  expectations,
+                  failedExpectations,
+                  result: health,
```

Comment:
Why this change from `health` to `result`?
We use `health` all around the frontend code.
Is it because the `predicate` function expects that?

Reply:
Yep that's correct, `predicate()` relies on `result` being present.